app/models/proxy.rb

```diff
@@ -408,21 +303,21 @@ def send_api_test_request!
 
     analytics.track 'Sandbox Proxy Test Request',
                     analytics_attributes.merge(
-                        success: success,
-                        uri: result.uri.to_s,
-                        status: result.code
+                      success: success,
+                      uri: result.uri.to_s,
+                      status: result.code
                     )
 
     success
   end
 
   def enabled
```

Comment:
I dunno if this suggestion is easy or not, but since it is expecting a boolean, the method should be called `enabled?`

Reply:
Better live that for later